projects/project-player-library/src/lib/shared/task-card/task-card.component.html

```diff
@@ -0,0 +1,65 @@
+<mat-card class="items-card">
+    <div>
+        <div class="items-header" >
+            <span (click)="moveToDetailsTask(task._id)">
+                        <span class="complete-task" *ngIf="task.status !== 'completed'"></span>
+                        <mat-icon *ngIf="task.status === 'completed'"  style="color: green;" fontIcon="check_circle"></mat-icon>
+                <h4 >
+                    {{task.name}}
+                </h4>
+            </span>
+            <div>
+                <button mat-icon-button [matMenuTriggerFor]="menu" aria-label="Example icon-button with a menu">
+                    <mat-icon color="primary">more_vert</mat-icon>
+                  </button>
+                  <mat-menu #menu="matMenu" class="menu">
+
+                    <button class="btn-menu" mat-menu-item style="padding: 0;" *ngIf="!submittedImprovement">
```

Comment:
Edit, Share, Delete options shouldn't be hardcoded. It should be coming from an array and we have loop it and show the options

Reply:
done.